TaskModules/powershell/TaskModuleIISManageUtility/AppCmdOnTargetMachines.ps1

```diff
@@ -82,16 +82,16 @@ function Test-BindingExist
         $site = $site.ToLower()
         if($site.Contains($siteName.ToLower()) -and $site.Contains($binding.ToLower()))
         {
-            Write-Verbose "Given binding already exists for the current website (`"$siteName`")."
+            Write-Verbose "Binding ($protocol / $ipAddress : $port : $hostname) already exists for the current website (`"$siteName`")."

```

Comment:
For consistency will you have the hostname, protocol etc same as how it is in inetmgr. Similar comment as given for you in the Task end.